sulong/tests/com.oracle.truffle.llvm.tests.debug/trace/testLoop.txt

```diff
@@ -0,0 +1,192 @@
+        MEMBER int "int" "j" "6"
+        MEMBER int "int" "i" "4"
+STOP 39 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "10"
+        MEMBER int "int" "i" "4"
+
+
+
+STOP 45 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "0"
+STOP 47 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "0"
+
+STOP 45 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "1"
+STOP 47 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "1"
+        MEMBER int "int" "i" "1"
+
+STOP 45 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "1"
+        MEMBER int "int" "i" "2"
+STOP 47 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "3"
+        MEMBER int "int" "i" "2"
+
+STOP 45 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "3"
+        MEMBER int "int" "i" "3"
+STOP 47 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "6"
+        MEMBER int "int" "i" "3"
+
+STOP 45 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "6"
+        MEMBER int "int" "i" "4"
+STOP 47 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "10"
+        MEMBER int "int" "i" "4"
+
+
+
+STOP 54 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "0"
+        MEMBER int "int" "k" "0"
+
+STOP 56 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "0"
+        MEMBER int "int" "k" "0"
+STOP 58 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "0"
+        MEMBER int "int" "k" "0"
+
+STOP 56 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "0"
+        MEMBER int "int" "i" "1"
+        MEMBER int "int" "k" "0"
+STOP 58 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "1"
+        MEMBER int "int" "i" "1"
+        MEMBER int "int" "k" "0"
+
+STOP 56 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "1"
+        MEMBER int "int" "i" "2"
+        MEMBER int "int" "k" "0"
+STOP 58 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "3"
+        MEMBER int "int" "i" "2"
+        MEMBER int "int" "k" "0"
+
+STOP 56 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "3"
+        MEMBER int "int" "i" "3"
+        MEMBER int "int" "k" "0"
+STOP 58 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "6"
+        MEMBER int "int" "i" "3"
+        MEMBER int "int" "k" "0"
+
+STOP 56 CONTINUE "start"
+    OPEN_SCOPE
+        MEMBER int "int" "j" "6"
+        MEMBER int "int" "i" "4"
+        MEMBER int "int" "k" "0"
+STOP 58 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "10"
+        MEMBER int "int" "i" "4"
+        MEMBER int "int" "k" "0"
+
+STOP 61 CONTINUE "start"
+    OPEN_SCOPE
+	MEMBER int "int" "j" "10"
+        MEMBER int "int" "i" "5"
+        MEMBER int "int" "k" "0"
+
+BREAK 65 CONTINUE "start"
```

Comment:
The C file for this test only has 61 lines, you cannot stop at line 65. Did you perhaps reformat the C code without changing the trace?